libindy/indy-wallet/src/storage/default/transaction.rs

```diff
@@ -75,8 +76,8 @@ impl<'conn> Transaction<'conn> {
             DropBehavior::Commit => self.commit_(),
             DropBehavior::Rollback => self.rollback_(),
             DropBehavior::Ignore => Ok(()),
-            DropBehavior::Panic => {
-                panic!("drop behaviour is set to panic".to_string());
+            _ => {
```

Comment:
Should be kept original, the original match covers all scenarios, `_` might be dangerous in case of future updates